client/webserver/site/src/js/settings.ts

```diff
@@ -80,12 +80,13 @@ export default class SettingsPage extends BasePage {
       const wallet = asset.wallet
       if (wallet) {
         const bondAsset = this.currentDEX.bondAssets[asset.symbol]
-        if (wallet.synced && wallet.balance.available > bondAsset.amount) {
+        const bondsFeeBuffer = await this.getBondsFeeBuffer(assetID, page.regAssetForm)
+        if (wallet.synced && wallet.balance.available >= 2 * bondAsset.amount + bondsFeeBuffer) {
           this.animateConfirmForm(page.regAssetForm)
           return
         }
-        const txFee = await this.getRegistrationTxFeeEstimate(assetID, page.regAssetForm)
-        this.walletWaitForm.setWallet(wallet, txFee)
+        this.walletWaitForm.setWallet(wallet, bondsFeeBuffer)
+        this.currentForm = page.walletWait
```

Comment:
Not really the solution for master (`this.currentForm`), but this PR is going to cherry-pick to release-v0.6